include/rppt_tensor_audio_augmentations.h

```diff
@@ -94,6 +94,9 @@ RppStatus rppt_to_decibels_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_
  * \retval RPP_ERROR* Unsuccessful completion.
  */
 RppStatus rppt_pre_emphasis_filter_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32s *srcLengthTensor, Rpp32f *coeffTensor, RpptAudioBorderType borderType, rppHandle_t rppHandle);
+#ifdef GPU_SUPPORT
+RppStatus rppt_pre_emphasis_filter_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, Rpp32s *srcLengthTensor, Rpp32f *coeffTensor, RpptAudioBorderType borderType, rppHandle_t rppHandle);
```

Comment:
Please add the description for HIP API too
I have added similar description for ToDecibels HIP API
https://github.com/sampath1117/rpp/blob/sr/to_decibels_hip/include/rppt_tensor_audio_augmentations.h#L83

In recent RPP changes, we are following this approach of adding description for both api's

Reply:
Done